Remove disabled flaky test for claims form keyboard nav

NB28VT opened the pull request:
## Summary

As per [this list](https://depo-platform-documentation.scrollhelp.site/developer-docs/currently-disallowed-cypress-test-specs-on-va-gov), this test has been automatically silenced from running during CI because it fails intermittently and needs to be rewritten. As of 11/06/23, merging will be blocked if this test isn't rewritten or removed.

We're opting to remove it for now; a [ticket](https://app.zenhub.com/workspaces/disability-benefits-experience-team-carbs-6470c8bfffee9809b2634a52/issues/gh/department-of-veterans-affairs/va.gov-team/68612) has been created to rewrite this test when we have time to dedicate to it:  


## Related issue(s)
- Ticket to rewrite this https://app.zenhub.com/workspaces/disability-benefits-experience-team-carbs-6470c8bfffee9809b2634a52/issues/gh/department-of-veterans-affairs/va.gov-team/68612

## Testing done

- This is a simple removal of a test.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._
N/A

## What areas of the site does it impact?

This is an important test and should be rewritten as it concerns Form 526 accessbility.

## Acceptance criteria

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [ X Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added (N/A)
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

My team opted to remove this test rather than comment it out or use the `skip` Cypress keyword; as the test will likely need a full rewrite we didn't see a reason to keep the file present at all. Is this consistent with what other teams have done?

NB28VT committed:
Remove disabled flaky test for claims form keyboard nav

As per https://depo-platform-documentation.scrollhelp.site/developer-docs/currently-disallowed-cypress-test-specs-on-va-gov, this has been automatically silenced from running during CI because it fails intermittently and needs to be rewritten. As of 11/06/23, merging will be blocked if this test isn't rewritten or removed.

We're opting to comment it for now; a ticket has been created to rewrite this test when we have time to dedicate to it:  https://app.zenhub.com/workspaces/disability-benefits-experience-team-carbs-6470c8bfffee9809b2634a52/issues/gh/department-of-veterans-affairs/va.gov-team/68612

NB28VT committed:
Merge branch 'main' into nb-6784-disable-flaky-test

tommasina-va reviewed (approved):
Thanks for doing this! 👏🏻

NB28VT committed:
Merge branch 'main' into nb-6784-disable-flaky-test

NB28VT committed:
Merge branch 'main' into nb-6784-disable-flaky-test

NB28VT committed:
Merge branch 'main' into nb-6784-disable-flaky-test

NB28VT committed:
Remove flaky All Claims test from codebase

This flaky test has an existing ticket to be fixed which is currently being worked on; as it is blocking merging for several other team members, we are opting to remove it entirely instead of commenting it out.

NB28VT commented:
There is an open PR to actually fix this
